src/main/resources/templates/home.html

```diff
@@ -11,78 +11,40 @@
       </th:block>
 
       <div class="grid-row">
-
-        <main class="column-two-thirds" id="main" role="main">
-          <h1 class="heading-large" th:utext="${friendlyRegisterName}"></h1>
-          <th:block th:utext="${registerText}"></th:block>
-          <p data-click-events="" data-click-category="Content" data-click-action="Button Clicked">
+        <div class="column-two-thirds">
+          <h1 class="heading-large">
+            <th:block th:utext="${friendlyRegisterName}"></th:block>
+            <span class="heading-secondary" th:utext="${registerText}"></span>
+          </h1>
+          <span data-click-events="" data-click-category="Content" data-click-action="Button Clicked">
             <a href="/records" class="button" role="button">View Register</a>
-          </p>
-          <p>The <span th:utext="${friendlyRegisterName}"></span> contains <span th:text="${totalRecords}"></span>
-            records and includes the following fields:</p>
-          <div th:replace="fragments/field-list.html::field-list(${fields})"></div>
+          </span>
+
+          <h2 class="heading-medium">Using this register</h2>
 
-          <h2>Using this register</h2>
+          <h3 class="heading-small">Base URL</h3>
+          <pre class="preview-box"><code class="language-markup" th:utext="${registerResolver.baseUriFor(registerName)}"></code></pre>
```

Comment:
This gives me the following when running `country` locally:

![screen shot 2018-02-21 at 12 05 19](https://user-images.githubusercontent.com/5422487/36479158-8f032228-16ff-11e8-89b9-5f3fd311e31f.png)

Reply:
I get the same but assumed it was being it was local?

Reply:
yes agree I think it's a problem unfortunately